portal/code/cx-portal/src/components/AuthProvider.tsx

```diff
@@ -0,0 +1,16 @@
+import { Context, useEffect } from 'react'
+import { ReactReduxContextValue, useDispatch } from 'react-redux'
+import { setLoggedUser } from 'state/features/user/userSlice'
+import { IUser } from 'types/user/UserTypes'
+
+export function AuthProvider(props: {
+  children: any
+  user: IUser
+  context?: Context<ReactReduxContextValue>
```

Comment:
As I saw, here is a usage of Context API. Is there any usage of this prop or any necessity  ?

Reply:
this is just a leftover of the example I copied - will remove this prop since it's not used here